project.clj

```diff
@@ -9,6 +9,7 @@
                  [http-kit "2.2.0"]
                  ;;logging
                  [com.taoensso/timbre "4.10.0"]
+                 [cheshire "5.8.1"]
```

Comment:
cheshire and clojure.data.json are libraries that do the same thing; json serialization. There's  no need to import both of them